core/src/main/java/org/elasticsearch/search/sort/ScriptSortParser.java

```diff
@@ -140,7 +140,6 @@ public SortField parse(XContentParser parser, QueryShardContext context) throws
             sortMode = reverse ? MultiValueMode.MAX : MultiValueMode.MIN;
         }
 
-        // If nested_path is specified, then wrap the `fieldComparatorSource` in a `NestedFieldComparatorSource`
```

Comment:
Oh, and NestedFieldComparatorSource doesn't seem to exist any more. Another sign that the comment might be outdated.